src/common/entity/alarm_panel_icon.ts

```diff
@@ -0,0 +1,21 @@
+/** Return an icon representing a alarm panel state. */
+
+export const alarmPanelIcon = (state?: string) => {
+  switch (state) {
+    case "armed_away":
+    case "armed_vacation":
+      return "hass:shield-lock";
```

Comment:
```suggestion
      return "hass:shield-sun";
```

Reply:
For vacation?

Reply:
Make sense now, when we use `moon` for night. Or we can use `mdi:shield-airplane`.

Reply:
Yeah airplane would work well too!

Reply:
@bramkragten Should I push the `airplane` icon? I can do it right now.

Reply:
yeah sure! :-)

Reply:
Done.